module/VuFind/src/VuFind/ILS/Logic/Holds.php

```diff
@@ -180,63 +180,59 @@ protected function formatHoldings($holdings)
      */
     public function getHoldings($id, $ids = null)
     {
-        $holdings = [];
+        if (!$this->catalog) {
+            return [];
+        }
+        // Retrieve stored patron credentials; it is the responsibility of the
+        // controller and view to inform the user that these credentials are
+        // needed for hold data.
+        try {
+            $patron = $this->ilsAuth->storedCatalogLogin();
+
+            // Does this ILS Driver handle consortial holdings?
+            $config = $this->catalog->checkFunction(
+                'Holds', compact('id', 'patron')
+            );
+        } catch (ILSException $e) {
+            $patron = false;
+            $config = [];
+        }
 
-        // Get Holdings Data
-        if ($this->catalog) {
-            // Retrieve stored patron credentials; it is the responsibility of the
-            // controller and view to inform the user that these credentials are
-            // needed for hold data.
-            try {
-                $patron = $this->ilsAuth->storedCatalogLogin();
-
-                // Does this ILS Driver handle consortial holdings?
-                $config = $this->catalog->checkFunction(
-                    'Holds', compact('id', 'patron')
-                );
-            } catch (ILSException $e) {
-                $patron = false;
-                $config = [];
-            }
+        if (isset($config['consortium']) && $config['consortium'] == true) {
+            $result = $this->catalog->getConsortialHoldings(
+                $id, $patron ? $patron : null, $ids
+            );
+        } else {
+            $result = $this->catalog
+                ->getHolding($id, $patron ? $patron : null, $options);
```

Comment:
$options appears to be undefined at this point and is never used again; is this a typo?

Reply:
Oops, how did I do that? It was supposed to be a parameter for the method. Now fixed.

Reply:
Thanks!